models/user.go

```diff
@@ -0,0 +1,64 @@
+package models
+
+import (
+	"errors"
+	"fmt"
+)
+
+type User struct {
```

Comment:
@zigara and now I have an uppercase type, what's the deal with that?

Reply:
If you want to be able to see the User type outside of the models package you have to make it public. If you were only using this type inside of the models package it could be private.